### ISSUE TITLE
Flags error when a [15:8] access occurs

### ISSUE THREAD

**Issue by JonathanSalwan** [JonathanSalwan/Triton#185]
```
[KO] 0x400584: or ah, 0x8 (1 error)
     Register       : pf
     Symbolic Value : 0000000000000001
     Concrete Value : 0000000000000000
     Expression     : (bvxor (bvxor (bvxor (bvxor (bvxor (bvxor (bvxor (bvxor (_ bv1 1) ((_ extract 0 0) (bvlshr ((_ extract 7 0) #162) (_ bv0 8)))) ((_ extract 0 0) (bvlshr ((_ extract 7 0) #162) (_ bv1 8)))) ((_ extract 0 0) (bvlshr ((_ extract 7 0) #162) (_ bv2 8)))) ((_ extract 0 0) (bvlshr ((_ extract 7 0) #162) (_ bv3 8)))) ((_ extract 0 0) (bvlshr ((_ extract 7 0) #162) (_ bv4 8)))) ((_ extract 0 0) (bvlshr ((_ extract 7 0) #162) (_ bv5 8)))) ((_ extract 0 0) (bvlshr ((_ extract 7 0) #162) (_ bv6 8)))) ((_ extract 0 0) (bvlshr ((_ extract 7 0) #162) (_ bv7 8))))
[OK] 0x400587: mov rax, 0xfbad2284
[KO] 0x400591: or ah, 0x8 (2 error)
     Register       : pf
     Symbolic Value : 0000000000000001
     Concrete Value : 0000000000000000
     Expression     : (bvxor (bvxor (bvxor (bvxor (bvxor (bvxor (bvxor (bvxor (_ bv1 1) ((_ extract 0 0) (bvlshr ((_ extract 7 0) #171) (_ bv0 8)))) ((_ extract 0 0) (bvlshr ((_ extract 7 0) #171) (_ bv1 8)))) ((_ extract 0 0) (bvlshr ((_ extract 7 0) #171) (_ bv2 8)))) ((_ extract 0 0) (bvlshr ((_ extract 7 0) #171) (_ bv3 8)))) ((_ extract 0 0) (bvlshr ((_ extract 7 0) #171) (_ bv4 8)))) ((_ extract 0 0) (bvlshr ((_ extract 7 0) #171) (_ bv5 8)))) ((_ extract 0 0) (bvlshr ((_ extract 7 0) #171) (_ bv6 8)))) ((_ extract 0 0) (bvlshr ((_ extract 7 0) #171) (_ bv7 8))))
     Register       : sf
     Symbolic Value : 0000000000000001
     Concrete Value : 0000000000000000
     Expression     : (ite (= ((_ extract 7 7) #171) (_ bv1 1)) (_ bv1 1) (_ bv0 1))
```

We must update **EflagsBuilder** and **EflagsExpressions** to take into account the vector size of the destination.

**Comment by JonathanSalwan** [comment]
Still not done in specific cases.

**Comment by JonathanSalwan** [comment]
It's because we use sometime `createRegSE()` and others times `createSE()`. Two solutions:

* `createSE()` should build expression as like as a register bitvector.
* `Eflags` should take the `high` and `low` in their parameters.

**Comment by JonathanSalwan** [comment]
Fixed with the v0.3 (#226)
